### DISCUSSION TITLE
Clases 30/08

### DISCUSSION THREAD

**Discussion by MayAmatista** [INGCOM-UNRN-P1/cursada-2023#117] [Preguntas y respuestas] [answered]
Buenas tardes, quería consultar si el día de hoy tenemos clases normalmente ya que Paola nos envió un mensaje en el cual indicaba que se suspendían debido a las condiciones climáticas, pero que de todas formas consultemos ya que quedaba supeditado a la decisión de cada docente.

Saludos!

**Comment by martinvilu** [comment, marked as answer]
La clase es normalmente virtual los miércoles, asi que no hay cambios.
